Paraphrenia/Assets/Scripts/LevelEvents/RandomFlicker.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class RandomFlicker : MonoBehaviour
+{
+    public UnityEvent OnFlickerOn;
+    public UnityEvent OnFlickerOff;
+    
+    [SerializeField] private float minOffDelay = 0.1f;
+    [SerializeField] private float maxOffDelay = 0.6f;
+    [SerializeField] private float minOnDelay = 0.02f;
+    [SerializeField] private float maxOnDelay = 0.1f;
+
+    private bool active;
+
+    void Start()
+    {
+        active = true;
+        StartCoroutine(FlickerTimer());
+    }
+    void OnApplicationQuit()
+    {
+        active = false;
+    }
+
+    IEnumerator FlickerTimer()
```

Comment:
Geef bij elke functie duidelijk aan of het private of public is, hier mist dus private